File: clase5/codigo_jgrasp/DLL/Node.java
public class Node {
    int data;
    Node next;
    Node prev;

    public Node() {
        this.data = 0;
        this.next = null;
        this.prev = null;
    }

    public Node(int data) {
        this.data = data;
        this.next = null;
        this.prev = null;
    }

    @Override
    public String toString() {
        return "[" + data + "]";
    }
}

File: clase6/linded_list/single_LL_double_end/python/node.py
class Node:
    def __init__(self, data=0):
        self.data = data
        self.next = None

    def __str__(self):
        return f"[{self.data}]"

if __name__ == "__main__":
    # Example usage
    node1 = Node(5)
    print(node1)  # Output: [5]
    node2 = Node()
    print(node2)  # Output: [0]

